src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -176,6 +176,11 @@ public void setPerson(Person target, Person editedPerson) {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public void recordCommand(String userInput) {
+            //This is a dummy test.
+        }
```

Comment:
I notice that there is a dummy method right here, I believe it is not called at all when the user invokes the ``HistoryCommand``, do you think it is better to remove it? 😄